CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ Release channels have their own copy of this changelog:
   * Unhide `--accounts-db-access-storages-method` for agave-validator and agave-ledger-tool and change default to `file`
   * Remove tracer stats from banking-trace. `banking-trace` directory should be cleared when restarting on v2.2 for first time. It will not break if not cleared, but the file will be a mix of new/old format. (#4043)
   * Add `--snapshot-zstd-compression-level` to set the compression level when archiving snapshots with zstd.
+  * Deprecate `--tower-storage` and all `--etcd-*` arguments
```

Comment:
I have a tentative approval to BP this to v2.2 given that we haven't actually had a v2.2 tag yet. Will ensure that a BP reviewer gives me a ship it on this review